chapter2/tests/t1/chapter2-t1-two-files.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -eo pipefail
+# Args check
```

Comment:
Please use this style guide for docs and variable names and update all docs in this file according to it: https://google.github.io/styleguide/shellguide.html#constants-and-environment-variable-names

Reply:
I don't see that you used this guide for functions docs https://google.github.io/styleguide/shellguide.html

Reply:
@rtest12 what about this?

Reply:
> @rtest12 what about this?

But this code is outdated, i fixed it before..